packages/devtools_app/lib/src/screens/memory/memory_graph_model.dart

```diff
@@ -147,19 +145,23 @@ HeapGraph convertHeapGraph(
       _monitorClasses.putIfAbsent(i, () => className);
     }
 
-    classes[i] = HeapGraphClassLive(c);
+    classesDraft[i] = HeapGraphClassLive(c);
   }
 
+  final classes = classesDraft as List<HeapGraphClassLive>;
```

Comment:
can you perform these casts directly in the `return HeapGraph` statement so we don't have to create a new var for classes, elements, and externals?

Reply:
done